app/views/registers/show.html.haml

```diff
@@ -1,5 +1,5 @@
-- @page_title = "#{@register.register_name} register"
-- @page_description = @register.register_description
+- @page_title = "#{@register.seo_title.present? ? @register.seo_title : @register.register_name + ' register'}"
+- @page_description = "#{@register.meta_description.present? ? @register.meta_description : @register.register_description}"
```

Comment:
I think we should also consider using this for the secondary heading span within the H1 as I don't think meta description alone will have a big SEO effect.

Reply:
The meta description is for google search only.